src/main/java/auto_ckz/domain/car/Car.java

```diff
@@ -0,0 +1,48 @@
+package auto_ckz.domain.car;
+
+import auto_ckz.domain.abstracts.AbstractEntity;
+import lombok.Data;
+import org.hibernate.validator.constraints.Length;
+
+import javax.persistence.Entity;
+import java.sql.Date;
+
+@Entity
+@Data
+public class Car extends AbstractEntity {
+
+    @Length(max = 50)
+    private String make;
+
+    @Length(max = 50)
+    private String model;
+
+    @Length(max = 4)
+    private int year;
```

Comment:
Czy `@Length` można stosować przy intach? http://docs.jboss.org/hibernate/stable/validator/api/index.html?org/hibernate/validator/constraints/Length.html

I czy nie lepiej byłoby trzymać to jako `String`?